htmlstuffblahblah/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
Don't add this please. Only push stuff to a pull request that is part of the final PR, so no temp work or placeholders.